ui/src/jobs/components/DateFromNow.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+class DateFromNow extends Component {
+  render() {
+    const { date, prefix } = this.props;
+    return <span>{`${prefix} ${moment(date).fromNow()}`}</span>;
+  }
+}
+
+DateFromNow.propTypes = {
+  date: PropTypes.string.isRequired,
+  prefix: PropTypes.string,
```

Comment:
why needs to be a prop, you could just render it outside `created <DateFromNow date={createdDate}/>`